authors/apps/articles/views.py

```diff
@@ -97,3 +97,55 @@ def destroy(self, request, slug):
             {"message": "Article is deleted"},
             status=status.HTTP_200_OK
         )
+
+
+class LikeArticleAPIView(generics.UpdateAPIView):
+    permission_classes = (IsAuthenticated,)
+    queryset = ArticleLikes.objects.all()
+
+    def get_object(self):
+        """Method to get an article"""
+        slug = self.kwargs.get("slug")
+        return get_object_or_404(Article, slug=slug)
+
+    def get_queryset(self, request):
+        """Method to get liked or disliked articles"""
+        article = self.get_object()
+        liked_article = self.queryset.filter(
+            article=article, user=request.user).first()
+        return liked_article
+
+    def save_like_status(self, message):
```

Comment:
@VivianDoreen please rename this to what it actually does. No saving of an object happens inside this method.
```suggestion
    def like_response_message(self, message):
```